ch.ivyteam.smart.core/src/ch/ivyteam/smart/core/schema/OpenAiSchemaMapper.java

```diff
@@ -47,7 +32,7 @@ public JsonNode optimize(ObjectNode schema) {
 
   private void staticSchemaRef(ObjectNode schemaRef) {
     schemaRef.remove("pattern");
-    schemaRef.put("const", target.toString());
+    schemaRef.put("const", path);
```

Comment:
I fear we can't just path any path in here (now pointing to the internal URL of the schema).
It should refer to official public schema: https://json-schema-axonivy.com/...